### PR TITLE
Test admin network page operations

### PR DESCRIPTION
**Pull request by gdyuldin**

### CODE REVIEW

**Comment by schipiga** [comment]
Pls, resolve conflicts

**Comment by gdyuldin** [comment]
Done

File: horizon_autotests/tests/test_networks.py [hunk 1]
```diff
@@ -58,6 +58,17 @@ def test_create_delete_subnet(self, networks_steps, network):
 
         networks_steps.admin_delete_subnet(network.name, subnet_name)
 
+    def test_create_edit_delete_port(self, horizon, networks_steps, network):
```
Comment (schipiga) [inline horizon_autotests/tests/test_networks.py]: `horizon` fixture usage is redundant, it's activated in `network`

File: horizon_autotests/tests/test_networks.py [hunk 2]
```diff
@@ -58,6 +58,17 @@ def test_create_delete_subnet(self, networks_steps, network):
...
+        """Checks create delete subnet functionality"""
```
Comment (schipiga) [inline horizon_autotests/tests/test_networks.py]: Seems, it must have another doc string

**Pull request merged**